smll/FaceDetector.cpp

```diff
@@ -47,6 +47,7 @@ namespace smll {
 		, m_camera_w(0)
 		, m_camera_h(0)
 		, isPrevInit(false)
+		, landmarks_inited(false)
 		, cropInfo(0,0,0,0) {
```

Comment:
I think there is a typo in landmarks_inited. Can you rename the variable.

Reply:
Sure